MapboxNavigation/Resources/Base.lproj/Localizable.strings

```diff
@@ -1,6 +1,9 @@
 ﻿/* Dismiss button title on the steps view */
 "DISMISS_STEPS_TITLE" = "Close";
 
+/* Unable to read instruction aloud */
+"FAILED_INSTRUCTION" = "Unable to read instruction aloud";
```

Comment:
A human-readable error message should read like a complete sentence, so it should end with a period.